ts/a11y/explorer.ts

```diff
@@ -240,17 +240,22 @@ export function ExplorerMathDocumentMixin<B extends MathDocumentConstructor<HTML
      */
     public static OPTIONS: OptionList = {
       ...BaseDocument.OPTIONS,
-      enrichSpeech: 'shallow',                   // overrides option in EnrichedMathDocument
       enableExplorer: true,
       renderActions: expandable({
         ...BaseDocument.OPTIONS.renderActions,
         explorable: [STATE.EXPLORER]
       }),
+      sre: expandable({
+        speech: 'shallow',                 // overrides option in EnrichedMathDocument
+        domain: 'mathspeak',               // speech rules domain
+        style: 'default',                  // speech rules style
+        locale: 'en'                       // switch the locale
+      }),
       a11y: {
         align: 'top',                      // placement of magnified expression
         backgroundColor: 'Blue',           // color for background of selected sub-expression
         backgroundOpacity: 20,             // opacity for background of selected sub-expression
-        braille: false,                    // switch on Braille output
+        brailleExp: false,                 // switch on Braille output
```

Comment:
Note that these changes in configuration options mean that some people's configurations will no longer be valid, and will begin to report errors when this update is made.  Since that is a breaking change, it should require a different version number than 3.1.3.  Of course, I doubt that many people have such options in their pages, but it might be nice to make this more backward compatible.  One possibility would be to use the `constructor()` below to do 

``` javascript
const a11y = (args[2] || {}).a11y;
if (a11y) {
  for (const option of ['braille', 'speech']) {
     if (a11y.hasOwnProperty(option)) {
       a11y[option + 'Exp'] = a11y[option];
       delete a11y[option];
     }
  }
}
```

just before the `super(...args)` so that old options will be converted to the new ones for now.  We can remove this in a future release.

Reply:
You are right, we should not introduce a breaking change. It is only necessary for the `speech` option anyway, so we can get away with a simple `if`.

Reply:
OK, sounds good.  (Is that because Braille is new in this version?)

Reply:
No, the `braille` option was always the switch for the Braille explorer. SRE does not have a `braille` option, but a `modality` option that takes `braille` as a value. I just wanted to make it uniform to `speechExp`.

Reply:
This was entirely unnecessary as it did not solve the actual problem anyway. That problem is related to the order of attached key explorers: the last stops event propagation. However, detaching and re-attaching explorers does not always keep the order of the events on the node in sync. So sometimes events are stopped too early.
A subsequent PR will implement the following:
* Explorers attach themselves only once by adding events on update of the document.
* They are only detached "virtually", i.e., just don't run.
* The last "virtually" attached explorer is `stoppable`.